scripts/unsupported/combine_tracks_postprocessing_cnv/combine_tracks.wdl

```diff
@@ -0,0 +1,621 @@
+    # WARNING: In the workflow, you should calculate the disk space as an input to this task (disk_space_gb).
+	Int default_disk_space_gb = 100
+
+    # Mem is in units of GB but our command and memory runtime values are in MB
+    Int machine_mem = if defined(mem_gb) then mem_gb *1000 else default_ram_mb
+    Int command_mem = machine_mem - 1000
+
+	command <<<
+	    
+        set -e
+        export GATK_LOCAL_JAR=${default="/root/gatk.jar" gatk4_jar_override}
+
+		echo "======= Germline Tagging "
+		gatk --java-options "-Xmx${command_mem}m" \
+			TagGermlineEvents \
+            --segments ${tumor_called_seg} --called-matched-normal-seg-file ${matched_normal_called_seg} \
+            -O ${output_name}.germline_tagged.seg -R ${ref_fasta} \
+            ${"--endpoint-padding " + germline_tagging_padding}
+
+        echo "======= Centromeres "
+    	gatk --java-options "-Xmx${command_mem}m" \
+    	     CombineSegmentBreakpoints \
+            --segments ${output_name}.germline_tagged.seg --segments ${centromere_tracks_seg}  \
+            --columns-of-interest ${sep=" --columns-of-interest " columns_of_interest} \
+            -O ${output_name}.centro.seg -R ${ref_fasta}
+
+        echo "======= GISTIC blacklist "
+    	gatk --java-options "-Xmx${command_mem}m" \
+    	     CombineSegmentBreakpoints \
+            --segments ${output_name}.centro.seg --segments ${gistic_blacklist_tracks_seg}  \
+            --columns-of-interest ${sep=" --columns-of-interest " columns_of_interest} \
+            --columns-of-interest ID \
+            -O ${output_name}.final.seg -R ${ref_fasta}
+	>>>
+
+    runtime {
+        docker: gatk_docker
+        memory: machine_mem + " MB"
+        # Note that the space before SSD and HDD should be included.
+        disks: "local-disk " + select_first([disk_space_gb, default_disk_space_gb]) + if use_ssd then " SSD" else " HDD"
+        preemptible: select_first([preemptible_attempts, 3])
+        cpu: select_first([cpu, 1])
+        bootDiskSizeGb: 20
+    }
+
+    output {
+        File germline_tagged_with_tracks_seg = "${output_name}.final.seg"
+        File germline_tagged_with_centro_track_seg = "${output_name}.centro.seg"
+        File germline_tagged_seg = "${output_name}.germline_tagged.seg"
+    }
+}
+
+task PruneGermlineTagged {
+    File germline_tagged_seg
+
+    String output_filename = basename(germline_tagged_seg) + ".pruned.seg"
+    command <<<
+    set -e
+        python <<EOF
+import pandas
+import os.path
+tumor_tagged = "${germline_tagged_seg}"
+
+tumor_tagged_df = pandas.read_csv(tumor_tagged, delimiter="\t", comment="@")
+tumor_tagged_pruned_df = tumor_tagged_df[(tumor_tagged_df["POSSIBLE_GERMLINE"] == "0") & (tumor_tagged_df["type"] != "centromere") & (tumor_tagged_df["ID"].isna())]
+output_filename = "${output_filename}"
+print(output_filename)
+tumor_tagged_pruned_df.to_csv(output_filename, sep="\t", index=False)
+
+EOF
+
+    >>>
+
+    runtime {
+        docker: "amancevice/pandas"
```

Comment:
Is it appropriate to use this Docker repo?  Seems like we should make our own image and put it in one of the Broad's GCR repos.